features/org.wso2.analytics.is.feature/src/main/capp/realtime/IdPPropertiesUpdateAnalyticsExecutionPlan/IsAnalytics-ExecutionPlan-IdPPropertiesUpdateData.siddhiql

```diff
@@ -0,0 +1,18 @@
+/* Enter a unique ExecutionPlan */
```

Comment:
Missing license header

Reply:
The contents of this file will be shown to the user. Therefore the license header is not generally added to the .siddhiql files as far as I know.